Documentation/Makefile

```diff
@@ -45,7 +45,32 @@ copy-api:
 	@$(ECHO_GEN)_api
 	$(QUIET)cp -r ../api _api
 
-epub latex html: builder-image copy-api
+HELM_VALUES = helm-values.rst
+
+HELM_DOCS_VERSION := v1.3.0
+HELM_DOCS_SHA := 12f065115654c9a1d6819b938487d327f1fdb5c32bdc2a5c74afef22ed68f5ca
+HELM_DOCS_IMAGE := docker.io/jnorwood/helm-docs:$(HELM_DOCS_VERSION)@sha256:$(HELM_DOCS_SHA)
+HELM_DOCS_ROOT_PATH := /src
```

Comment:
I had to change `/src` to something else, because `DOCKER_CTR` already maps to `/src`, and for me it ended up in:
```
docker: Error response from daemon: Duplicate mount point: /src.
```